v4/export/sql.go

```diff
@@ -120,30 +124,51 @@ func SelectAllFromTable(conf *Config, db *sql.DB, database, table string) (Table
 	}, nil
 }
 
-func buildSelectAllQuery(conf *Config, db *sql.DB, database, table string) (string, error) {
+func buildSelectAllQuery(conf *Config, db *sql.DB, database, table string, where string, orderByClause string) (string, error) {
 	var query strings.Builder
+	var err error
 	query.WriteString("SELECT * FROM ")
 	query.WriteString(database)
 	query.WriteString(".")
 	query.WriteString(table)
+
+	separator := "WHERE"
+	if conf.Where != "" {
```

Comment:
why not merge  `where condition` in the outside of this function, make code simpler